projects/storefrontapp-e2e-cypress/cypress/support/storage.commands.ts

```diff
@@ -24,6 +24,17 @@ declare namespace Cypress {
         ```
        */
     restoreLocalStorage: () => void;
+    /**
+       * Reset the local storage memory.
+       *
+       * @memberof Cypress.Chainable
+       *
+       * @example
+        ```
+        cy.resetLocalStorageMemory()
+        ```
+       */
+    clearLocalStorageMemory: () => void;
```

Comment:
[Cypress clears local storage between tests](https://docs.cypress.io/api/commands/clearlocalstorage.html#Syntax). Is this necessary?

Reply:
Yes, because this does not clear the browser's local storage, this clears the memory where we store data to be able to restore the browser's local storage.